Calculator/Calculator/Model/Calculator.swift

```diff
@@ -27,55 +36,27 @@ class Calculator {
         case "×":
             return first * second
         case "÷":
+            try checkDividedNumber(second: second)
             return first / second
         default:
-            return 10000000
+            throw CalculatorError.invalidOperator
         }
     }
     
-    func returnCalculationResult() -> String {
-        for item in postfix.postfix {
+    func returnCalculationResult(postfix: [String]) throws -> String? {
+        for item in postfix {
             if isNumberInPostfix(item: item) {
-                numberStack.append(item)
+                numberStack.push(item: item)
             }else {
-                let second = numberStack.removeLast()
-                let first = numberStack.removeLast()
-                let calculateResult = calculatePostfix(first: first, second: second, arithmethicOperator: item)
-                numberStack.append(String(calculateResult))
+                let second = numberStack.pop()
+                let first = numberStack.pop()
+                guard let first = first, let second = second else {
+                    throw CalculatorError.invalidNumber
+                }
```

Comment:
아래와 같이 줄여볼 수 있을 것 같아요.
```suggestion
                guard let second = numberStack.pop(),
                      let first = numberStack.pop() else {
                    throw CalculatorError.invalidNumber
                }
```